PublicationsRouter/v2/sword-in/USER.md

```diff
@@ -17,81 +17,64 @@ but you won't need it much after that.
 The Service Document lists the collections that you can send your notifications to, the mimetypes and the packaging formats
 that are supported.
 
-In this version of the system there are only two collections:
+There will only be one collection - this will be the collection for your user account.
 
-1. validate - you can send content here to check whether it's correctly formatted for sending to the router
-2. notify - you can send content here when you're ready to notify us of a new publication
+You can get the URL for your collection by retrieving the service document, which you can do like this:
 
-You'll be able to get the URLs for those collections by retrieving the service document, which you can do like this:
+    GET /
 
-    GET /service-document
-
-This will require you to provide your username and api key via HTTP Basic Authentication.
+This will require you to provide your email and password via HTTP Basic Authentication.
 
 For example, using curl, you might do the following:
 
-    curl -i http://username:api_key@pubsrouter.jisc.ac.uk/sword/service-document
+    curl -i https://email:password@pubrouter.jisc.ac.uk/sword/
 
 (check with Jisc for the correct base url of the live service).
 
-## Send a package for validation
-
-Once you have constructed a Zip file of the appropriate packaging format (see the JPER packaging documentation for details),
-you'll want to check that it works with the router.  You can do this by sending it to the "validate" collection before you
-try sending it for real.
-
-Get the URL of the collection from the Service Document (in the section above), and then do the following
-
-    POST /collection/validate
-    Content-Disposition: filename=filename.zip
-    Content-Type: application/zip
-    Packaging: https://pubrouter.jisc.ac.uk/FilesAndJATS
-    
-    [binary content]
-
-This will require you to provide your username and api key via HTTP Basic Authentication.
-
-For example, using curl, you might do the following:
-
-    curl -i --data-binary "@test.zip" -H "Content-Disposition: filename=test.zip" -H "Content-Type: application/zip" -H "Packaging: https://pubsrouter.jisc.ac.uk/FilesAndJATS" http://userame:api_key@pubsrouter.jisc.ac.uk/sword/collection/validate
-
-In response to this you will either receive a SWORDv2 error document detailing any problems with the request, or a
-successful 202 (Accepted) response.  If you get the latter, you are good to start sending your notifications to the "notify"
-collection.
-
 ## Send a package as a new notification
 
 Once you have constructed and validated (see above) a  Zip file of the appropriate packaging format (see the JPER packaging documentation for details),
-you can send your notifications to the live "notify" collection for inclusion in the router.
+you can send your notifications to your collection.
```

Comment:
As Ben said: I'd suggest linking the JPER packaging docs here, it's got it's own .md I think